### DISCUSSION TITLE
style和tag两个插件没有办法共用

### DISCUSSION THREAD

**Discussion by Frozen1009** [MuiseDestiny/zotero-style#1084] [Q&A]
macbook刚刚升级了zotero7，之前用6的时候也有这个问题，需要style的阅读进度条和tag的标签，并且tag的标签是在文章标题最前面的，现在打开zotero之后会闪一下有tag的界面，然后变成只有style的进度条的界面。在style的上一个版本中是正常的，但是也有bug，窗口不能关闭，只能小化，关闭之后再打开是没有阅读进度条

**Comment by egg-1-1** [comment]
你好这两个插件怎么下载 ？有途径吗十分感谢
